src/test/java/com/solr/clientwrapper/rest/SolrSchemaResourceTest.java

```diff
@@ -128,8 +128,7 @@ void testDeleteSchema() throws Exception {
 		SolrSchemaDTO solrSchemaDTO = new SolrSchemaDTO(tableName, name, attributes);
 		setMockitoBadResponseForService();
 		restAMockMvc.perform(MockMvcRequestBuilders.delete(solrendpoint + "/delete/" + tableName + "/" + name)
-				.contentType(MediaType.APPLICATION_JSON).content(TestUtil.convertObjectToJsonBytes(solrSchemaDTO)))
-				.andExpect(status().isBadRequest());
```

Comment:
These codes were not supposed to be altered, was running fine when I created this. Any reason you had to remove that part?

Reply:
@shubhamphepale Why have you commented .andExpect(...)